Userland/Libraries/LibGUI/VimEditingEngine.cpp

```diff
@@ -971,13 +971,22 @@ bool VimEditingEngine::on_key_in_normal_mode(const KeyEvent& event)
                 move_one_up(event);
                 switch_to_insert_mode();
                 return true;
-            // FIXME: Integrate these into vim motions too.
-            case (KeyCode::Key_LeftBrace):
-                move_to_previous_empty_lines_block();
+            case (KeyCode::Key_LeftBrace): {
+                auto amount = m_motion.amount() > 0 ? m_motion.amount() : 1;
+                m_motion.reset();
+                for (int i = 0; i < amount; i++) {
+                    move_to_previous_empty_lines_block();
+                }
```

Comment:
please omit curly braces for single-line if/for/while etc. bodies

Reply:
Updated.